.readme-partials.yaml

```diff
@@ -5,6 +5,48 @@ custom_content: |
   
   The Storage Control API creates one space to perform metadata-specific, control plane, and long-running operations apart from the Storage API. Separating these operations from the Storage API improves API standardization and lets you run faster releases.
   
+  If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file:
+  ```xml
+  <dependencyManagement>
+      <dependencies>
+          <dependency>
+              <groupId>com.google.cloud</groupId>
+              <artifactId>libraries-bom</artifactId>
+              <version>26.37.0</version>
+              <type>pom</type>
+              <scope>import</scope>
+          </dependency>
+      </dependencies>
+  </dependencyManagement>
+  
+  <dependencies>
+      <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage-control</artifactId>
+  </dependency>
+  ```
+  
+  If you are using Maven without the BOM, add this to your dependencies:
+  <!-- {x-version-update-start:google-cloud-storage:released} -->
+  ```xml
+  <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage-control</artifactId>
+      <version>2.36.1-alpha</version><!-- {x-version-update:google-cloud-storage:current} -->
```

Comment:
These versions appearing in README should use `{x-version-update:google-cloud-storage:released}` instead of `{x-version-update:google-cloud-storage:current}`, because library users do not use "-SNAPSHOT" versions.